src/styles/_catalog.scss

```diff
@@ -14,9 +14,22 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: $translation-time;
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  &__card:hover {
+    transform: scale(120%, 120%) ;
+    color: #34568b;
+    border: 3px solid black;
+    border-radius: 10px;
+    font-size: 20px;
+    background-color: lightgray;
+    cursor: pointer;
```

Comment:
replace cursor: pointer from :hover to card, it will work the same ( Mentor asked me the same) :)